modules/Microsoft.Web/sites/deploy.bicep

```
@@ -356,6 +363,16 @@ module app_slots 'slots/deploy.bicep' = [for (slot, index) in slots: {
   }
 }]
 
+//[for (basicPublishingCredentialsPolicy, index) in basicPublishingCredentialsPolicies:
+
+module app_basicPublishingCredentialsPolicies 'basicPublishingCredentialsPolicies/deploy.bicep' = {
+  name: '${uniqueString(deployment().name, location)}-Site-Publis-Cred' //-${index}'
+  params: {
+    webAppName: app.name
+    name: basicPublishingCredentialsPolicyName
+  }
```

Missing telemetry inheritance for added child
```suggestion
  enableDefaultTelemetry: enableReferencedModulesTelemetry
  }
```